### ISSUE TITLE
Inconsistent, mangled, outdated posts

### ISSUE THREAD

**Issue by nearmax** [NEAR-DevHub/neardevhub-bos#649]
**Describe the bug**
Posts are missing from the feed. The reposts to near.social have inconsistent format. There are three issues that might be related:

**Issue 1**
**To Reproduce**
1. Go to https://near.social/devgovgigs.near/widget/Post?id=2923 observe the post 
2. Go to https://near.social/devhub.near/widget/app?page=feed&recency=all observe that the post is missing

**Expected behavior**
The post is present in the list of all posts.

**Issue 2**
**To Reproduce**
1. Go to https://near.social/devgovgigs.near/widget/Post?id=2918 observe the post 
2. Go to https://near.social/devhub.near/widget/app?page=feed&recency=all observe that the post is missing

**Expected behavior**
The post is present in the list of all posts.

**Issue 3**
I go to my near.social feed and observe that some posts are displayed well while some posts are just raw JSON:

<img width="901" alt="image" src="https://github.com/NEAR-DevHub/neardevhub-bos/assets/35039879/0fccb1b6-e6cc-4cd6-b9b1-dcc6837e8708">
<img width="877" alt="image" src="https://github.com/NEAR-DevHub/neardevhub-bos/assets/35039879/c205275f-352c-42d7-9669-8c679c37fab7">
I then follow the link and the post is still raw JSON: https://near.social/devgovgigs.near/widget/Post?id=2920

**Comment by frol** [comment]
@nearmax Just wanted to let you know that, I have extracted the third issue into #650, so we can address it separately, as I don't think that they are related.

@ailisp Do we have an issue with QueryAPI again?

**Comment by ailisp** [comment]
Yes looks like a queryapi issue. It doesn't pick up new blocks since January 24, 2024 at 9:13 AM . There is no infra level of log on why it is stopped and queryapi dashboard still show it as running: https://near.org/dataplatform.near/widget/QueryApi.App?selectedIndexerPath=bo.near/devhub_v36

I'll request Query API team for help looking why it silently stopped

**Comment by ailisp** [comment]
This issue is fixed by QueryAPI team restarting on their side. I can confirm new posts are appeared. From Darun Seethammagari in QueryAPI team:

> Yeah it seems like it isn't running. I'm investigating what the cause of it shutting down might be. In the meantime, I will restart Runner in prod and see if that gets the indexer going again. We'll put a fix in as soon as we can for the cause, once found.
Hey Bo. The restart seems to have resolved the issue for now. I see your indexer running, and it had a small backlog of 50 matching blocks, all of which have been processed.
...
 I was not able to glean why Devhub-36 specifically failed. The logs for DevHub_v36 end abruptly without any associated error. However, other indexers had also crashed unexpectedly in the days before and after. From what I can tell it's related to the way we make calls Hasura during DmlHandler init which may not be caught. It's a mystery how the error managed to dodge all of the try catches in its way though. I'll continue investigating any other indexer crashing contributors.
